### DISCUSSION TITLE
Building AAB for Android Using Flash Builder and AIR SDK Flex not working

### DISCUSSION THREAD

**Discussion by ebernste123** [airsdk/Adobe-Runtime-Support#3206] [Q&A]
I built an application using Flex and Adobe Flash Builder. I have updated the AIRSDK_Flex to 50.2.5.1. Flash Builder builds an APK, but it is only a 32-bit version (armv7). I am trying to build both armv7 and armv8 and bundle them together. I have modified the adt.cfg file and uncommented the line: 
  CreateAndroidAppBundle=true

Nothing I do builds an Android App Bundle (.aab) or a 64-bit apk. The current apk that is always built is correctly built, but Google will not accept non 64-bit APKs, 

What am I doing wrong ?

**Comment by ajwfrost** [comment]
Hi

If you've got that `CreateAndroidAppBundle=true` set up properly - in the adt.cfg file that's in your ~/.airsdk folder (on Windows, c:\users\username\.airsdk\adt.cfg) - then it should be overriding the armv7 request that Flash Builder is providing, and creating an AAB file, even though it will likely be named as an .apk file still...

If that's not working then it may be Flash Builder isn't using the correct path to build with the updated compiler; if you also set the `DebugOut=true` in the config file, you should be seeing a file "adt.log" in the home folder, or if you launch the AIR SDK Manager and go to the Troubleshooting tab, you should see the build information appearing when you build in Flash Builder.

The output - in the log file or in the troubleshooting tab - should help to see what's actually happening. It may be we can't do the Gradle-based building because of a missing dependency, or if your app descriptor namespace version is too low, or similar...

thanks
